exchanges/request/mock_test.go

```diff
@@ -0,0 +1,27 @@
+package request
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMockResponse(t *testing.T) {
```

Comment:
I will strongly encourage you write tests at a function level and not just a coverage level in a OneBigTest. Helps enforce rules and expectations especially when tests/functions are updated